WNPRC_EHR/resources/schemas/dbscripts/postgresql/wnprc-22.001-22.002.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE IF EXISTS wnprc.animal_requests RENAME COLUMN principalinvestigator TO principalinvestigatorold;
```

Comment:
do we need to include information about this changes in the metadata for the module